vipps-ecom-api.md

```diff
@@ -761,64 +745,69 @@ public class UrlValidate {
 
 ## Callbacks
 
-Callbacks allow Vipps to send the payment order details to the merchant.
-This can offer a faster user experience than polling, but you cannot rely on callbacks alone. You must also use poling as described in the [Polling guidelines](#polling-guidelines).
-
-The request body contains different information depending on whether the payment type is set to
-`"eComm Regular Payment"` or `"eComm Express Payment"`:
-
-* `"eComm Regular Payment"`: The callback contains the order and transaction details.
-* `"eComm Express Payment"`: The callback contains the order and transaction details
-  *and in addition* the user details and shipping details.
-
-The callback will be performed *only once* during a payment process, for one of the
-following events:
+Callbacks allow Vipps to send the merchant information about the payment.
+Callback are made for the following events:
 
 * Payment successful
 * Payment failed
 * Payment rejected
 * Payment timed out
 
-Vipps makes *one* attempt at a callback and cannot guarantee that it succeeds,
-as it depends on network, firewalls and other factors that Vipps cannot control.
-
-If the communication is broken during the process for some reason and Vipps
-is not able to execute callback to the merchant's server, the callback will
-not be retried.
-
-**Important:** Vipps callbacks require a response within 3 seconds; otherwise, they will time out.
-
-If you do not receive any confirmation on the payment request, you *must* call
-[`GET:/ecomm/v2/payments/{orderId}/details`](https://vippsas.github.io/vipps-developer-docs/api/ecom#tag/Vipps-eCom-API/operation/getPaymentDetailsUsingGET) to get the status of the payment.
-
-**Please note:** Do not rely only on callbacks alone, but also poll the `/details` endpoint
-as described in
+**Please note:** Callback offer a faster user experience than polling, but you
+cannot rely on callbacks alone. You must also poll
```

Comment:
Suggest we say `should not`.
`cannot rely on callbacks alone.` is a bit strong for my tastes.